apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -70,6 +70,9 @@ export const askAITutor = createAsyncThunk(
     } else {
       systemPrompt = generalChatSystemPrompt;
     }
+    if (state.aiTutor.level?.levelSpecificPrompt) {
+      systemPrompt += state.aiTutor.level?.levelSpecificPrompt;
+    }
```

Comment:
@Erin007 Based on the discussion we had during our AI Tutor sync, I decided to add this prompt to the end of all the system prompts instead of overriding just the general chat prompt like outlined in the jira task. It seemed like more inputs to the model was the direction we are currently heading.

Reply:
Let's double check the spacing here. If the systemPrompt and levelSpecificPrompt are smashed together it might be hard for the AI to interpret.